libraries/exoplayer/src/main/java/androidx/media3/exoplayer/drm/KeyLoadInfo.java

```diff
@@ -9,18 +10,48 @@
  * to complete loading a DRM key
  */
 public class KeyLoadInfo {
-  public LoadEventInfo loadEventInfo;
-  public final List<LoadEventInfo> retriedLoadRequests;
 
-  public KeyLoadInfo() {
-    retriedLoadRequests = new ArrayList<>();
-  }
+  public static class Builder {
+    private LoadEventInfo loadEventInfo;
+    private final List<LoadEventInfo> retriedLoadRequests;
+    private final List<SchemeData> schemeDatas;
+
+    public Builder(List<SchemeData> schemeDatas) {
+      this.schemeDatas = schemeDatas;
+      retriedLoadRequests = new ArrayList<>();
+      loadEventInfo = null;
```

Comment:
Also, in the renamed version handled the case where `schemeDatas` can be null (the caller has a Nullable member here).